docs/source/release_notes/release-notes.rst

```diff
@@ -60,25 +78,25 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2397>`__)
 
     *    - |fixed|
-         - Oracle auto-shrink is now not enabled by default. This is an experimental feature to allow Oracle non-EE users to compact automatically.
-           However, it has seen timeouts for large amounts of data. We are turning this off by default, until we figure out a better retry mechanism for
-           shrink failures.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/ABCD>`__)
+         - Oracle auto-shrink is now disabled by default.
+           This is an experimental feature allowing Oracle non-EE users to compact automatically.
+           We decided to turn it off by default since we have observed timeouts for large amounts of data, until we find a better retry mechanism for shrink failures.
```

Comment:
"we find a better retry mechanism for shrink failures"
- is this tracked by a github issue? Would be cool to link to it here.

Reply:
@hsaraogi for creating the ticket, if still relevant ^